File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/dart/practicas_3/informacion_basica_ciclo.dart
import 'dart:io';

void main () {

  print('Ingrese la cantidad de usuarios a los que quiere saber su informacion:');
  int cuantos = int.parse(stdin.readLineSync()!);

  for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

    print('Nombres:');
    String name = stdin.readLineSync()!;

    print('Apellidos:');
    String lastname = stdin.readLineSync()!;

    print('Edad:');
    int age = int.parse(stdin.readLineSync()!);

    print('Tipo documento:');
    String type = stdin.readLineSync()!;

    print('Numero de documento:');
    String document = stdin.readLineSync()!;

    print('Numero de telefono:');
    String number = stdin.readLineSync()!;

    print('Estado civil:');
    String state = stdin.readLineSync()!;

    print('Lugar donde vive:');
    String place = stdin.readLineSync()!;

    print('----------------------------------------------------------');

    print('INFORMACION ORGANIZADA NUMERO: $vueltas');

    print('----------------------------------------------------------');

    print('Nombres: $name');
    print('Apellidos: $lastname');
    print('Edad: $age');
    print('Tipo documento: $type');
    print('Numero de documento: $document');
    print('Numero de telefono: $number');
    print('Estado civil: $state');
    print('Lugar donde vive: $place');

    print('----------------------------------------------------------');

  }

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/ruby/practicas_3/informacion_basica_ciclo.rb

puts "Ingrese la cantidad de usuarios a los que quiere saber su informacion:"
cuantos = gets.chomp.to_i

for vueltas in 1..cuantos

    puts "Nombres:"
    name = gets.chomp

    puts "Apellidos:"
    lastname = gets.chomp

    puts "Edad:"
    age = gets.chomp.to_i

    puts "Tipo documento:"
    type = gets.chomp

    puts "Numero de documento:"
    document = gets.chomp

    puts "Numero de telefono:"
    number = gets.chomp

    puts "Estado civil:"
    state = gets.chomp

    puts "Lugar donde vive:"
    place = gets.chomp

    puts "----------------------------------------------------------"

    puts "INFORMACION ORGANIZADA NUMERO: #{vueltas}"

    puts "----------------------------------------------------------"

    puts "Nombres: #{name}"
    puts "Apellidos: #{lastname}"
    puts "Tipo documento: #{type}"
    puts "Numero de documento: #{document}"
    puts "Numero de telefono: #{number}"
    puts "Estado civil: #{state}"
    puts "Lugar donde vive: #{place}"

    puts "----------------------------------------------------------"
end